packages/@o3r/schematics/src/utility/ast.ts

```diff
@@ -225,3 +225,66 @@ export const sortClassElement = (classElement1: ts.ClassElement, classElement2:
     }
   }
 };
+
+/**
+ * Transformer to be used to fix the string literals generated by creating a new SourceFile (for instance, using #generateClassElementsFromString)
+ *
+ * @param ctx
+ */
+export const fixStringLiterals: ts.TransformerFactory<ts.SourceFile> = (ctx) => (rootNode) => {
+  const { factory } = ctx;
+  const visit = (node: ts.Node): ts.Node => {
+    if (node.kind === ts.SyntaxKind.StringLiteral) {
+      return factory.createStringLiteral(node.getText().replace(/^["'](.*)["']$/, '$1'), true);
```

Comment:
what happens if we have a string lie the following: "Test 'Hello' worlds". Is that a valid scenario for this method?

Reply:
it'll be generated as `'Test \'Hello\' worlds'`